base_layer/wallet/src/output_manager_service/recovery/standard_outputs_recoverer.rs

```diff
@@ -265,37 +274,26 @@ where
     /// seen so far.
     async fn update_outputs_script_private_key_and_update_key_manager_index(
         &mut self,
-        output: &mut WalletOutput,
+        output: &WalletOutput,
```

Comment:
The function has changed, so the comment and name must reflect what is happening inside.

Reply:
I removed the function, it has the potential to be very slow, and its not needed